my-app/src/Square.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Square(props) {
```

Comment:
One small thing - you can use object de-structuring if you like to avoid having to say `props.` in the return of the function. 

```js
function Square({id, onClick, value}){
```